pkgs/tools/graphics/pikchr/default.nix

```diff
@@ -1,6 +1,9 @@
 { lib
 , stdenv
 , fetchurl
+, tcl
+
```

Comment:
```suggestion
```

Reply:
I separated the parameter from the inputs on purpose, I'd like to keep it like this if possible

Reply:
As you wish.